LBHackney-IT/lbh-fss-portal-frontend
# Feature/resend user authentication

**Description**

Users that have been added by an admin will receive an email containing authentication details. This task is to add a button to allow admins to resend authentication details, this button should not be visible if the user has already authenticated their account or was not invited by an admin (handled by the 'unverified' user status) 


**Test instruction**

- pull this branch 
- navigate to the **/users** page
- click on the first user in the list 'Adah Johnson', this user does not not have an  'unverified' status and therefore the 'Resend authentication details' button should not be displayed
- search for 'Jaylin Rutherford Sr.', this user does have an  'unverified' status and therefore the 'Resend authentication details' button should be displayed
- click the 'Resend authentication details' to see it returns a successful response and redirects to the **/users** page

![image](https://user-images.githubusercontent.com/69238887/95333374-9a38bb00-08a4-11eb-8d9d-4467d3a956d7.png)


**Ticket**

[not clear which ticket this task falls under]
